definitions/npm/react-navigation_v1.x.x/flow_v0.60.x-/react-navigation_v1.x.x.js

```diff
@@ -0,0 +1,1013 @@
+// @flow
+
+import * as React from 'react';
+import { Animated, Image } from 'react-native';
+import type {
+  StyleObj,
+} from 'react-native/Libraries/StyleSheet/StyleSheetTypes';
+import type { Layout } from 'react-native-tab-view/src/TabViewTypeDefinitions';
```

Comment:
Are any of these imports problematic? Things seem to work, but in a conversation on Slack @calebmer seemed to indicate that it shouldn't be possible to import from packages like this.